File: SoloLearn Codes/Cheer Creator/cheer_creator.cpp
#include <iostream>
using namespace std;

int main() {
    int x;
    cin >> x;
    if (x < 1) cout << "shh";
    else if (x > 10) cout << "High Five";
    else for (int i = 0; i < x; i++) cout << "Ra!";
    return 0;
}

File: SoloLearn Codes/Cheer Creator/CheerCreator.java
import java.util.Scanner;

public class CheerCreator
{
    public static void main(String[] args) {
        int yard = new Scanner(System.in).nextInt();
        if (yard < 1) System.out.println("shh");
        else if (yard > 10) System.out.println("High Five");
        else System.out.println("Ra!".repeat(yard));
	}
}